utilities/output_tools.py

```diff
@@ -87,6 +87,8 @@ def write_analysis_output(results, outfile, args):
         to_append.append(args.postfix)
     if args.maxFiles > 0:
         to_append.append(f"maxFiles{args.maxFiles}")
+    if args.postfix:
+        to_append.append(args.customTag)
```

Comment:
this line should probably changed to "to_append.append(args.postfix)"

Reply:
right, I corrected it